src/features/defi/providers/foxy/components/FoxyManager/Overview/Claim/ClaimStatus.tsx

```diff
@@ -100,12 +97,22 @@ export const ClaimStatus: React.FC<ClaimStatusProps> = ({ accountId }) => {
 
   const feeMarketData = useAppSelector(state => selectMarketDataById(state, feeAssetId))
 
-  const accountFilter = useMemo(() => ({ accountId: accountId ?? '' }), [accountId])
-  const bip44Params = useAppSelector(state => selectBIP44ParamsByAccountId(state, accountFilter))
+  const dispatch = useAppDispatch()
+  // TODO: maybeRefetchOpportunities heuristics
+  const refetchFoxyBalances = useCallback(() => {
+    dispatch(
+      opportunitiesApi.endpoints.getOpportunitiesUserData.initiate(
+        {
+          accountId: accountId ?? '',
```

Comment:
early return vs fetching this with an effectively invalid accountId?

Reply:
9e0abab06602a84f07ab5ec4ebb44a47f19458f8